connections/pyproject.toml

```diff
@@ -24,7 +24,7 @@ pytest-ruff = "^0.4.1"
 black = "~24.4.2"
 
 [tool.poetry.group.integration.dependencies]
-aries-askar = { version = "~0.3.2" }
+aries-askar = { version = "~0.4.2" }
```

Comment:
Update to 0.4.3 already, or hold that to update them all together?

Reply:
I think I'll update it to 0.4.3 after acapy main gets the change, because it's pointing to main. The rest can wait until the next acapy release.

I'm not entirely sure why this is here though. This was done a long time ago. If the plugin doesn't explicitly use askar independently I don't think this should be needed. I'll try and look into removing this integration group for all plugins.